hhnk_threedi_tools/core/folders_hydamo_validation.py

```diff
@@ -0,0 +1,46 @@
+"""Folderstructuur voor de hydamo validatie."""
+import os
+from dataclasses import dataclass
+from pathlib import Path
+from typing import Union
+
+import hhnk_research_tools as hrt
+
+@dataclass
+class SourcePaths:
+    """Locaties van inputbestanden voor het valideren van de hydamo"""
+
+    dem_file: Union[str, Path, hrt.Raster]
+    hydamo_file: str
+    rules_file: str
+
+    def __post_init__(self):
+        """
+        dem_file(str): raster source van DEM
+        hydamo_file (str): path to hydamo
+        rules_file (str): path to rules
+        """
+
+        self.dem = hrt.Raster(self.dem_file)
+        self.hydamo = hrt.File(self.hydamo_file)
+        self.rules = hrt.File(self.rules_file)
+        self.verify()
+
+    def verify(self):
```

Comment:
Ik heb in hrt een PR klaarstaan https://github.com/HHNK/hhnk-research-tools/pull/61 waarmee je dit niet op elke class erbij hoeft te zetten.

gebruik; `hrt.Folder.verify_exists(cls=self, keys=["dem", "hydamo", "rules"])`